projects/TG-AST190036.yaml

```
@@ -0,0 +1,13 @@
+Name: TG-AST190036 
+Description: Exo-Cartography: Constraining Planet Formation through Mapping the Three-Dimensional Architectures of Planetary Systems
```

```suggestion
Description: "Exo-Cartography: Constraining Planet Formation through Mapping the Three-Dimensional Architectures of Planetary Systems"
```
To handle the `:` in the description